.github/workflows/go.yml

```diff
@@ -17,11 +17,10 @@ jobs:
   build:
     runs-on: ubuntu-latest
     steps:
-
       - name: golang
         uses: actions/setup-go@v2
         with:
-          go-version: 1.18
+          go-version: 1.19
```

Comment:
I guess this can be changed to 1.20 as well.

Reply:
oh yes, missed that one.
done now